applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -228,14 +263,25 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 	int ret;
 	uint8_t channel_index;
 
-	ret = headset_conn_index_get(stream->conn, &channel_index);
+	ret = channel_index_get(stream->conn, NULL, &channel_index);
 	if (ret) {
 		LOG_ERR("Stream not found");
 		return;
 	}
 
+	if (stream->ep->dir == BT_AUDIO_DIR_SINK) {
+		headsets[channel_index].sink_stream = stream;
+		LOG_INF("%s sink stream connected", headsets[channel_index].ch_name);
+	} else if (stream->ep->dir == BT_AUDIO_DIR_SOURCE) {
+		headsets[channel_index].source_stream = stream;
+		LOG_INF("%s source stream connected", headsets[channel_index].ch_name);
+	} else {
+		LOG_WRN("Dir not recognized");
```

Comment:
Probably using we can make it a little bit clear, like "End-point direction not recognized"